src/main.ts

```diff
@@ -58,6 +58,15 @@ async function execCommand(command: string, failingExitCode = 1): Promise<ExecRe
   return await p;
 }
 
+function scrubSecrets(input: string): string {
+  let output = input;
+  const authTokenMatches = input.match(/--auth-token=([\w.\S]+)/);
```

Comment:
Nice! Though this does make me wonder whether we need to obfuscate k8s secrets too?

Reply:
Example? K8s secrets won't be checked into this repo... The diff etc output I've seen is only
1) the commands run in the Action, which include the auth-token (`argocd app diff --auth-token=***`)
2) the YAML generated here from `helm template` and the like.

Reply:
Right, can't `helm template` output contain secrets?

Reply:
I don't think so, since it's purely based on what's checked into the repo.